typescript/infra/scripts/funding/reclaim-from-igp.ts

```diff
@@ -0,0 +1,41 @@
+import { AbacusCore, objMap, promiseObjAll } from '@abacus-network/sdk';
+
+import { getEnvironment, getEnvironmentConfig } from '../utils';
+
+async function main() {
+  const environment = await getEnvironment();
+  const coreConfig = await getEnvironmentConfig();
+  const multiProvider = await coreConfig.getMultiProvider();
+  const core: AbacusCore<any> = AbacusCore.fromEnvironment(
+    environment,
+    multiProvider,
+  );
+
+  const paymasters = core.map(
+    (_, contracts) => contracts.interchainGasPaymaster,
+  );
+
+  const balances = await promiseObjAll(
+    multiProvider.map((chain, chainConnection) => {
+      const provider = chainConnection.provider;
+      const paymasterAddress = paymasters[chain].address;
+      return provider.getBalance(paymasterAddress);
+    }),
+  );
+
+  console.log('Balances', balances);
+
+  const reclaimTxHashes = await promiseObjAll(
```

Comment:
should we require a minimum balance in order to claim from a chain? Just could see something like ethereum not having a large balance and us wasting gas. I don't think it necessarily needs to be per-chain, could be some value like `0.1` or something